src/autopas/tuning/utils/Smoothing.h

```diff
@@ -14,6 +14,35 @@
 
 namespace autopas::smoothing {
 
+/**
+ * Calculates the weights for the k-neighbors of the last point in the vector using the tri-cube function.
+ * If the sum of weights is <= 0 all neighbors are at the same x coordinate as the point and smoothing will not change
+ * the point. This case is indicated by the bool in the return tuple.
+ *
+ * @param points Sorted collection of observations.
+ * @param pointsPerEstimation Number of neighbors to consider for smoothing.
+ * @param maxDistFromIntervalEdge Largest distance between the point that shall be fitted and its neighbors.
+ * @return Tuple of a vector containing the weights for the neighbors
+ * and a bool indicating whether fitting is unnecessary.
```

Comment:
Can you clarify the wording here a bit? `true` means unnecessary?

Reply:
`true` means the fit is "ok". I'll add this when I move the declarations back.

Reply:
See b3e45a23